docs/02-guides/server-side-events/create-an-event.mdx

```diff
@@ -7,9 +7,119 @@ sidebar_position: 1
 
 <p>{frontMatter.description}</p>
 
-:::info WIP
+## What is an event emitter
 
-- Create an event emitter
-- Emit an event from your code
+The events are dispatched with what we call Event Emitter. The role of the event emitter is to push events to the Server event pipeline to be processed by the server.
```

Comment:
**issue:** I think your prettier isn't working, this line looks awfully long to me.